R/gcloud-utils.R

```diff
@@ -41,12 +41,15 @@ scope_deployment <- function(application = getwd(), config) {
 
   # default excludes plus any additional excludes in the config file
   config <- cloudml::config(config = config)
-  exclude <- c("local", "jobs")
+  exclude <- c("local", "jobs", ".git", ".svn")
```

Comment:
These regexs only apply to the top level listing so just the root .svn and .git folders (subfolders that include version control repos should be rare and in any case can be excluded if necessary).